### ISSUE TITLE
Dropdown: Works poorly in NVDA

### ISSUE THREAD

**Issue by splashdust** [sebgroup/green#863]
### Bug already reported ?

- [X] I confirm that I have checked if the bug already has been reported

### Component name

Dropdown

### Description

NVDA fails to announce the names of options in the dropdown. It might be because there are two nested `[role=listbox]` elements in there, but needs further investigation.

**Comment by splashdust** [comment]
Completed in #1016
